duckduckgo/macos-browser
# Option to add new notes or edit existing is disabled

Task/Issue URL: https://app.asana.com/0/1177771139624306/1201819261413662/f
Tech Design URL: -

**Description**:
As a first step to deprecate notes, we have decided to disable the ability to add new notes or edit existing notes.

**Steps to test this PR**:
1. Before you run this branch, make sure you add bunch of notes into Logins+
1. Checkout this branch, run the app and make sure you can't add a new note or edit existing one

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
